src/impact-dashboard/ImpactProjectionTableContainer.tsx

```diff
@@ -126,7 +126,6 @@ export function buildStaffData({
     buildTableRowFromCurves(staff, "Cases", countCasesForDay),
     Object.assign(
       buildTableRowFromCurves(staff, "Unable to work", countUnableToWorkForDay),
-      { overall: null },
```

Comment:
Can you briefly explain why this is removed?

Reply:
This was my bad! I thought this was part of the issue with the overall columns, but we do want this line--added it back in.